2017-10-29 Satvik Anand

* timespec.h (timespec_cmp): Changed function to use conditionals and not subtraction

--- timespec.h	2005-09-21 23:05:39.000000000 -0700
+++ temptimespec.h	2017-10-29 21:44:10.606712000 -0700
@@ -45,8 +45,9 @@
 static inline int
 timespec_cmp (struct timespec a, struct timespec b)
 {
-  int diff = a.tv_sec - b.tv_sec;
-  return diff ? diff : a.tv_nsec - b.tv_nsec;
+  if (a.tv_sec < b.tv_sec) return -1;
+  else if (b.tv_sec < a.tv_sec) return 1;
+  return 0;
 }
 
 # if ! HAVE_DECL_NANOSLEEP